modules/test/playwright/tests/journal-web/journalArticle.spec.ts

```diff
@@ -279,6 +279,61 @@ autoSaveAsDraftTest(
 	}
 );
 
+baseTest(
+	'LPD-31427: select web content display template with the Preview feature',
+	async ({journalEditArticlePage, page, site}) => {
+		page.on('dialog', (dialog) => dialog.accept());
+
+		await journalEditArticlePage.goto({siteUrl: site.friendlyUrlPath});
+
+		const title = getRandomString();
+
+		await page.getByText('Content', {exact: true}).waitFor();
```

Comment:
There are a few uses of this sentence `await page.getByText('Content', {exact: true}).waitFor();` in the test that I don't think are needed. Is there a reason?

Reply:
The journal editor is a bit slower to render and there was an inconsistency between runs without the waits. I did not want to create a flat 200-300 sec wait so I made the waiter based on the HTML editor which is the slowest component on the editor page. It appears repeatedly because if there are multiple reloads the page goes through the test.

Reply:
I see, but I still think you don't need the full page to be loaded , just the elements you are looking for. You already have other waitFor() like for Clear button, and those makes sense. 
I have run your test without those `await page.getByText('Content', {exact: true}).waitFor();` and get the same result, which fails for me by the way 😭